src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/ForgotPassword.cshtml

```diff
@@ -0,0 +1,25 @@
+﻿@model NHSD.BuyingCatalogue.Identity.Api.ViewModels.Account.ForgotPasswordViewModel
+
+<div class="nhsuk-width-container">
+    <main class="nhsuk-main-wrapper">
+        <div data-test-id="login-page">
+            <div class="nhsuk-grid-row">
+                <div class="nhsuk-grid-column-two-thirds">
+                    <h1 data-test-id="page-title" class="nhsuk-heading-xl nhsuk-u-margin-bottom-2">Enter your email address</h1>
+                    <h2 data-test-id="page-description" class="nhsuk-heading-s nhsuk-u-margin-bottom-5">You'll be sent a link to the address you provide so you can choose a password.</h2>
+                </div>
+            </div>
+            <div class="nhsuk-grid-row">
+                <div class="nhsuk-grid-column-two-thirds">
+                    <div data-test-id="error-summary" nhs-validation-summary title="There is a problem"></div>
+                    <form data-test-id="input-form" asp-controller="Account" asp-action="ForgotPassword" novalidate method="post" class="nhsuk-form-group">
+                        <div nhs-validation-input asp-for="EmailAddress" input-data-test-id="input-email-address" error-data-test-id="error-email-address" field-data-test-id="field-email-address"></div>
```

Comment:
`error-data-test-id` and `field-data-test-id` is this a thing? I haven't seen it before

Reply:
They're a cool hip Razor thing, if you delve into the TagHelper classes I've implemented you'll understand